modules/i18n/src/translations/en-US/portals/policy-administration.ts

```diff
@@ -85,6 +85,32 @@ export const policyAdministration: policyAdministrationNS = {
     },
     policyAlgorithm: {
         actionText: "Select a policy combining algorithm",
+        algorithmOptions: {
+            denyOverrides: {
+                description: "The deny overrides combining algorithm is intended for those cases where a deny decision should have priority over a permit decision."
```

Comment:
"Deny" decision will have priority over a "Permit" decision.

Reply:
According to the screenshot. the hint describes the algorithm type selected from the dropdown. Therefore, in the hint, let's keep the algorithm name similar to what appears in the dropdown.

```suggestion
                description: "The deny-overrides combining algorithm is intended for those cases where a deny decision should have priority over a permit decision."
```